.gitignore

```diff
@@ -130,6 +130,3 @@ dmypy.json
 
 # Plugin configuration file
 configuration.json
-
-# test generated files
-qp_klp/tests/*
```

Comment:
Why is this line being removed again?

Reply:
I believe it's to prevent example output files generated by the notebooks from being accidentally added to the repo. No worries on my end though. I can approve it as is.